common/hashicorp-vault/values.yaml

```diff
@@ -12,9 +12,30 @@ vault:
     extraEnvironmentVars:
       VAULT_CACERT: /var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt
       VAULT_ADDR: https://vault.vault.svc.cluster.local:8200
+    sgx: true
```

Comment:
I don't think we can turn this on for all patterns.

@mhjacks can you help figure out the best way to set this from the hub values file please?

Reply:
Yes. Broadly I've proposed using an `extraValueFiles` entry for these.